submissions/stanleyMubanga/database2/database.sql

```diff
@@ -17,7 +17,7 @@ CREATE TABLE passengers_details (
     passengers_id INT NOT NULL,
```

Comment:
I am wondering how I overlooked this. The passengers table should be merged with the passengers_details table.